### PR TITLE
Update readme with configuration description;

### PR DESCRIPTION
**Pull request by Antondomashnev**
Just an update in README.md about new way to configure the plugin.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 ### 0.3.0 (Next)
 
 * Your contribution here.
-* [#20](https://github.com/dblock/danger-changelog/pull/21): Added support for custom '* Your contribution here.' line - [@antondomashnev](https://github.com/antondomashnev).
+* [#25](https://github.com/dblock/danger-changelog/pull/25): Add information how to configure the plugin in README.md - [@antondomashnev](https://github.com/antondomashnev).
```
Comment (dblock) [inline CHANGELOG.md]: Typically I don't include README updates in CHANGELOG.

Comment (Antondomashnev) [inline CHANGELOG.md]: It makes sense, but we don't have any check in Danger to skip readme changes, do we?

Comment (dblock) [inline CHANGELOG.md]: Sounds like a feature request ;)

Comment (Antondomashnev) [inline CHANGELOG.md]: Already have one https://github.com/dblock/danger-changelog/issues/6

**Pull request merged**